Dockerfile

```diff
@@ -28,4 +32,6 @@ COPY provider_container_settings.json /analyzer-lsp/provider_settings.json
 
 WORKDIR /analyzer-lsp
 
-ENTRYPOINT ["konveyor-analyzer"]
+EXPOSE 5775/udp 6831/udp 6832/udp 5778 16686 14268 9411
```

Comment:
@Parthiba-Hazra thank you for incorporating the changes. One last thing, I missed this earlier, sorry. Can we only expose the 16686 port, as all other ports are only used internally?

Reply:
done :+1: